src/platforms/css.ts

```diff
@@ -7,10 +7,10 @@ const getCssSelectors = (outputFile: string): {selector: string; selectorLight:
   const lastSlash = outputFile.lastIndexOf('/')
   const outputBasename = outputFile.substring(lastSlash + 1, outputFile.indexOf('.'))
   const themeName = outputBasename.replace(/-/g, '_')
-  const mode = outputBasename.substring(0, 4) === 'dark' ? 'dark' : 'light'
+  // const mode = outputBasename.substring(0, 4) === 'dark' ? 'dark' : 'light'
 
   return {
-    selector: `[data-color-mode="${mode}"][data-${mode}-theme="${themeName}"]`,
+    selector: `[data-color-mode="light"][data-light-theme="${themeName}"], [data-color-mode="dark"][data-dark-theme="${themeName}"]`,
```

Comment:
Why are you changing this?

Reply:
If this is really the case, I will do a PR just for this, so that it is not hidden in this giant one and can be more easily tracked.

However I am not sure this is actually correct. Maybe @colebemis can add to this discussion.
In the current version we get the following:

```css
[data-color-mode="light"][data-light-theme="light_colorblind"],
[data-color-mode="auto"][data-light-theme="light_colorblind"] {

or

[data-color-mode="dark"][data-dark-theme="dark_high_contrast"],
[data-color-mode="auto"][data-light-theme="dark_high_contrast"] {
```
With the proposed changes we get:

```css
[data-color-mode="light"][data-light-theme="light_colorblind"],
[data-color-mode="dark"][data-dark-theme="light_colorblind"],
[data-color-mode="auto"][data-light-theme="light_colorblind"] {

or

[data-color-mode="light"][data-light-theme="dark_high_contrast"],
[data-color-mode="dark"][data-dark-theme="dark_high_contrast"],
[data-color-mode="auto"][data-light-theme="dark_high_contrast"] {
```

What is added is a line I think will never execute. As far as I see you can't get a combination of `data-color-mode="dark"` and a light theme like `data-dark-theme="light_colorblind"`. The only way you can get a light theme in _dark mode_ is using `auto`.

Am I missing something?

Reply:
> What is added is a line I think will never execute. As far as I see you can't get a combination of `data-color-mode="dark"` and a light theme like `data-dark-theme="light_colorblind"`. The only way you can get a light theme in _dark mode_ is using `auto`.
> 
> Am I missing something?

Yeah that's correct, in dotcom ui, when selecting single mode, you'll only get the corresponding `data-color-mode` to the stored theme value. The mismatch only happens if someone selects the opposite theme for there light/dark auto themes.

Reply:
Removing this change for now, and we can bring it back later if we find it necessary in dotcom!